fv3config/_datastore.py

```diff
@@ -51,6 +51,8 @@ def get_orographic_forcing_directory(config):
     if "orographic_forcing" not in config:
         raise ConfigError("config dictionary must have an 'orographic_forcing' key")
     parent_dirname = config["orographic_forcing"]
+    if not len(parent_dirname):
+        return ""
```

Comment:
If you make the change above, you won't need this code anymore because this should only ever be a string if this function is called.

```suggestion
```